core/src/fnc/type.rs

```diff
@@ -128,6 +131,93 @@ pub fn thing((arg1, arg2): (Value, Option<Value>)) -> Result<Value, Error> {
 	})
 }
 
+pub fn range(args: Vec<Value>) -> Result<Value, Error> {
+	if args.len() > 4 || args.is_empty() {
+		return Err(Error::InvalidArguments {
+			name: "type::range".to_owned(),
+			message: "Expected atleast 1 and at most 4 arguments".to_owned(),
+		});
+	}
+	let mut args = args.into_iter();
+
+	// Unwrap will never trigger since length is checked above.
+	let id = args.next().unwrap().as_string();
+	let start = args.next().and_then(|x| match x {
+		Value::Thing(v) => Some(v.id),
+		Value::Array(v) => Some(v.into()),
+		Value::Object(v) => Some(v.into()),
+		Value::Number(v) => Some(v.into()),
+		Value::Null => None,
+		v => Some(Id::from(v.as_string())),
```

Comment:
I assume this will take care of something like:
```sql
person:tobie..jaime
```

Like
```sql
type::range('person', 'tobie', 'jaime')
```

Is that correct?

Reply:
Yes, it is basically the same as to how `type::thing('person','tobie')` works.